wfsim/strax_interface.py

```diff
@@ -400,8 +400,8 @@ def set_config(self,):
             self.config.update(overrides)
 
         # Update gains to the nT defaults
-        self.to_pe = straxen.get_to_pe(self.run_id, self.config['gain_model'],
-                                       self.config['channel_map']['tpc'][1]+1)
+        self.to_pe = straxen.get_correction_from_cmt(self.run_id,
+                               self.config['gain_model'])
```

Comment:
Actually, this is the only change we really require for WFSim to run with straxen 0.19.0, the rest is just changed conventions for the testing.